tests/tinderbox_scraper/test_tinderbox_scraper.py

```diff
@@ -142,6 +142,20 @@
      'target': 'mozilla-central-firefox-25.0a1.en-US.linux-i686.txt',
      'target_url': 'firefox/tinderbox-builds/mozilla-central-linux/'
                    '1374583608/firefox-25.0a1.en-US.linux-i686.txt'},
+    # -a firefox -t tinderbox -p win32 --extension=txt
+    {'args': {'application': 'firefox',
+              'extension': 'txt',
+              'platform': 'win32'},
+     'target': 'mozilla-central-firefox-39.0a1.en-US.win32.txt',
```

Comment:
I missed to explicitly mention this but for the version you are using here please stick with 25.0a1 as all the other tests. You would only have to add the necessary txt file to the already existent folders. If we start to create new folders for each and every test it will become very confusing soon.